CMakeLists.txt

```diff
@@ -260,7 +260,7 @@ endif()
 # HIP support is enabled by default if not cross compiling. Note: a HIP-compatible
 # GPU with drivers is still required to actually run HIP workloads.
 set(IREE_HAL_DRIVER_HIP_DEFAULT ${IREE_HAL_DRIVER_DEFAULTS})
-if(CMAKE_CROSSCOMPILING)
+if(APPLE OR CMAKE_CROSSCOMPILING)
```

Comment:
Note that we also have https://github.com/iree-org/iree/pull/17996 pending which will flip both CUDA and HIP to default off, regardless of platform.

cc @maxbartel

Reply:
Oh okay. I'm fine either way.

Reply:
That PR landed. Closing this one.